engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -113,8 +112,10 @@ public ConfigurationImpl(ValidationProvider<?> provider) {
 	private ConfigurationImpl() {
 		this.validationBootstrapParameters = new ValidationBootstrapParameters();
 		TypeResolutionHelper typeResolutionHelper = new TypeResolutionHelper();
-		if ( isJavaFxInClasspath() ) {
-			validatedValueHandlers.add( new JavaFXPropertyValueUnwrapper( typeResolutionHelper ) );
+		ClassLoader classLoader = getClass().getClassLoader();
+		if ( isJavaFxInClasspath( classLoader ) ) {
+			JavaFXPropertyValueUnwrapper jfxUnwrapper = createJavaFXUnwrapperClass( typeResolutionHelper, classLoader );
```

Comment:
Would we not try to avoid the explicit use of `JavaFXPropertyValueUnwrapper`? Just use `ValidatedValueUnwrapper`!?